tests-e2e/ocw-ci-test-course/course-info.spec.ts

```diff
@@ -14,3 +17,23 @@ test("Course info section can be toggled", async ({ page }) => {
   await button.click()
   await expect(heading).toBeVisible()
 })
+
+test("Toggling topics does not affect drawer layout", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  await course.goto("/pages/section-1")
+
+  const heading = await page.getByRole("heading", { name: "Course Info" })
+  const topicCollapseButton = await page.getByRole("button", {
+    name: "Engineering subtopics"
+  })
+
+  await expect(heading).toBeVisible()
+  await expect(topicCollapseButton).toHaveAttribute("aria-expanded", "true")
+
+  await topicCollapseButton.click()
+  const mainSection = await page.locator(`#${MAIN_COURSE_SECTION_ID}`)
+  const drawer = await page.locator(`#${DESKTOP_COURSE_DRAWER_ID}`)
+
+  await expect(mainSection).toHaveClass(/.*col-lg-9.*/)
+  await expect(drawer).toHaveClass(/.*col-3.*/)
```

Comment:
This is for https://github.com/mitodl/ocw-hugo-themes/issues/1237#issuecomment-1700920199, right? 

Another option would be to compare `drawer.getBoundingClientRect().width` before / after the click, i guess.

Reply:
Yes. I tried it. In that approach, we only need to check the x-coordinates before and after. Checking the classes felt more relevant to the bug.